package.json

```diff
@@ -63,7 +64,9 @@
         "react": "18.2.0",
         "react-dom": "18.2.0",
         "react-is": "18.0.0",
+        "react-redux": "^8.0.4",
         "react-router-dom": "6.4.1",
+        "redux": "^4.2.0",
```

Comment:
Sweet!  got to this faster than I'd expected...

Reply:
But I need to install this on main and then bubble it through.